### PR TITLE
Support markdown in posts and store congress member pictures in app

### PR DESCRIPTION
**Pull request by TheFloatingString**

### CODE REVIEW

**Comment by TheFloatingString** [comment]
Bad pull request, have to reset package-lock.json to the contents on branch master

**Comment by TheFloatingString** [comment]
Need a way to only merge CongressPlaceholder and README

File: README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+which should "clone" (download) the repository
+
+then type:
+
+`cd msu-frontend` to enter the folder.
+
+Congratulations! You are in!
+
+
+## Project Structure
```
Comment (cherryman) [inline README.md]: Much of the project structure is self evident, and should be familiar with anyone who's used React Native.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Now, this is the cool part. Open cmd (Windows) or Terminal (MacOs or Linux).
+
+Go to Desktop (or any given directory you want), which you can navigate by using:
+
+* `dir` or `ls` which displays all the files in your current folder
+* `pwd` to print the path of your current folder
+* `cd <folder name>` to go to a new folder whose name is in `<folder name>`
+* `cd ..` to go back one folder
+
```
Comment (cherryman) [inline README.md]: Don't need to provide a basic shell tutorial in every repository that requires the tiniest bit of terminal usage. Anyone doing this sort of stuff will already know this.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+## Installing the Right Tools
+
+This project will NOT work if you do not install Git, NodeJS/npm or React Native.
+
+Scary? Not at all, just follow the instructions on the websites below to install the two tools.
+
+### Git (https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+Used for version control, simply just keeping track of progress
+
+### NodeJS and npm (https://www.npmjs.com/get-npm)
```
Comment (cherryman) [inline README.md]: Anyone familiar with React Native should know these steps. Not necessary.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+* `.gitignore`: folders and files that git ignores
+* `app.json`: Metadata (information) about the React Native app
+* `babel.config.js`: configurations for Babel (JavaScript compiler)
+* `index.js`: React Native file that connects App.js to the native mobile language
+* `metro.config.js`: configurations for Metro (design language)
+* `package-lock.json`: JSON file that records the exact versions of dependencies
+
+## Running the App Locally
+
+### Installing React Native dependencies
```
Comment (cherryman) [inline README.md]: Already documented in the React Native documentation, and one of the first few things someone does when learning how to configure React Native. At most, you should link directly to that documentation so people know where to start looking to get up and running. No need need to duplicate documentation.

**Pull request merged**